support/Environments/ocean_clang.sh

```diff
@@ -75,7 +75,6 @@ spectre_run_cmake() {
           -D CMAKE_C_COMPILER=clang \
           -D CMAKE_CXX_COMPILER=clang++ \
           -D CMAKE_Fortran_COMPILER=${GCC_HOME}/gfortran \
-          -D MEMORY_ALLOCATOR=SYSTEM \
```

Comment:
Are you sure you want to remove this line? I've seen a lot of troubles on HPC systems when not using the system allocator

Reply:
Yep! I've been building my BBH execs for weeks with no problems without that line